unime/src/routes/(app)/me/settings/about/+page.svelte

```diff
@@ -3,24 +3,47 @@
 
   import { TopNavBar } from '$lib/components';
   import { HeartFillIcon } from '$lib/icons';
+  import { state } from '$lib/stores';
+
+  import type { PageData } from './$types';
+
+  export let data: PageData;
 </script>
 
 <TopNavBar on:back={() => history.back()} title={$LL.SETTINGS.SUPPORT.ABOUT.NAVBAR_TITLE()} />
+
 <div class="content-height flex flex-col bg-silver dark:bg-navy">
-  <div class="flex flex-col p-8">
-    <!-- Footer -->
-    <div
-      class="flex flex-col items-center pt-4 text-[13px]/[24px] font-normal text-slate-500 opacity-50 dark:text-slate-300"
-    >
-      <!-- <div class="pb-4" /> -->
+  <h1 class="hidden">{$LL.SETTINGS.SUPPORT.ABOUT.TITLE}</h1>
```

Comment:
Why does this have a class "hidden"?

Reply:
Also, `TITLE` should be `TITLE()` to resolve from `typesafe-i18n`.

Reply:
Route did not have `h1`. But should always have `h1` for accessiblity.

Changed to `sr-only`. `hidden` is indeed ignored by screen readers.